lib/tasks/csv_load.rake

```diff
@@ -0,0 +1,11 @@
+namespace :csv_load do
```

Comment:
So this is how we'll be able to call all six of the csvs? Interested to learn how this works, but that's what I'm seeing when I look at it, hah.